### PR TITLE
Wadeea react homework week4

### PR DESCRIPTION
**Pull request by WadeeaKiwan**

### CODE REVIEW

File: homework-submission/react-hyf/react-hyf-client/src/Components/Navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.nav-list {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+}
+
+.nav-item {
+  border: 0.2px solid rgb(95, 94, 94);
+  padding: 15px;
```
Comment (Meno-101) [inline homework-submission/react-hyf/react-hyf-client/src/Components/Navbar/Navbar.css]: I would make the padding 15px on the `nav-link` for better experience: 

ezgif com-video-to-gif(2)

File: homework-submission/react-hyf/react-hyf-client/src/Components/Navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ */
+const Navbar = () => {
+  return (
+    <nav>
+      <ul className="nav-list">
+        <li className="nav-item">
+          <Link className="nav-link" to="/">
+            Home
+          </Link>
+        </li>{' '}
```
[the remark this answers is not in the document]
Comment (WadeeaKiwan) [reply homework-submission/react-hyf/react-hyf-client/src/Components/Navbar/Navbar.js]: Thanks... Done!

File: homework-submission/react-hyf/react-hyf-client/src/Components/Navbar/Navbar.css (hunk 1 above)
Comment (WadeeaKiwan) [reply homework-submission/react-hyf/react-hyf-client/src/Components/Navbar/Navbar.css]: Many thanks... Done

**Comment by Meno-101** [comment]
Thanks for the update, I will approve it again :)

**Comment by WadeeaKiwan** [comment]
> Hello @WadeeaKiwan,
> 
> **Well done, very neat clean homework!**
> 
> Very glad to see that, No changes will be requested, No comments are even a minor parts, feel free to change or add anything you would like.
> 
> And lastly good luck in the test and then the project 🎉

Many thanks @Meno-101 for your time, all advices and feedbacks all over the React Module... It was very nice to know you even (virtually) ;)

**Pull request closed without merging**